github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -7,7 +7,7 @@ presubmits:
     optional: false
     decorate: true
     decoration_config:
-      timeout: 1h
+      timeout: 2h
```

Comment:
2h would be enough ? 
maybe 2.5 h

Reply:
Creating the cluster is 20 minutes tops, Kubevirt SRIOV tests is 1h tops
I will increase it to 3 hours just in case